layer.md

```diff
@@ -156,3 +156,13 @@ Note that this opaque file will apply to _all_ children, including sub-directori
 Implementations SHOULD generate layers using _explicit whiteout_ files, but MUST accept both.
 
 Any given image is likely to be composed of several of these Image Filesystem Changeset tar archives.
+
+# Foreign Layers
+
+Certain layers, due to legal requirements, may not be regularly distributable.
+Typically, one can download such layers but they should never be uploaded.
```

Comment:
@RobDolinMS is going to quibble with this non-SHOULD should (e.g. opencontainers/runtime-spec#536 ;).  How about dropping this line, since the idea is covered in the rest of this section?